packages/core/test/multistep-dialog/multistepDialogTests.tsx

```diff
@@ -200,6 +200,31 @@ describe("<MultistepDialog>", () => {
         assert.strictEqual(dialog.find(NEXT_BUTTON).prop("disabled"), true);
         dialog.unmount();
     });
+
+    it("disables next for second step when disabled on nextButtonProps is set to true", () => {
+        const getNextButtonProps = (stepId: DialogStepId) => {
+            if (stepId === "two") return { disabled: true };
+
+            return {};
+        };
+
+        const dialog = mount(
+            <MultistepDialog nextButtonProps={getNextButtonProps} isOpen={true} usePortal={false}>
+                <DialogStep id="one" title="Step 1" panel={<Panel />} />
+                <DialogStep id="two" title="Step 2" panel={<Panel />} />
```

Comment:
hmm, on second thought, I think a better API would look like this:

```
<DialogStep id="two" title="Step 2" panel={<Panel />} nextButtonProps={{ disabled: true }} />
```

since we're calling `this.getDialogStepChildren()` and accessing each DialogStep's props while rendering the buttons anyway... we can make the API more straightforward and ergonomic this way

Reply:
Done